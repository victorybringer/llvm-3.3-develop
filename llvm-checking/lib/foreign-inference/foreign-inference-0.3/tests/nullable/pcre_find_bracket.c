extern int x;
char* pcre_find_bracket(char *code) {
  for(;;) {
    int c = *code;

    if(c == 0) return 0;

    if(c == 100) return code;

    code += x;
  }
}
