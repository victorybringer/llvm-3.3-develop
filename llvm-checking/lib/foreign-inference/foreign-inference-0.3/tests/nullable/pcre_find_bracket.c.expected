fromList [ ("@pcre_find_bracket", fromList ["%code"]) ]
